nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config

```diff
@@ -99,35 +99,18 @@ manifest {
   version = '{{ cookiecutter.version }}'
 }
 
-// Function to ensure that resource requirements don't go beyond
-// a maximum limit
-def check_max(obj, type) {
-  if(type == 'memory'){
+// Return the minimum between requirements and a maximum limit to ensure that resource requirements don't go over
```

Comment:
```suggestion
// Function to check that you dont request more compute resources than are available on your system 
```

Reply:
What don't you like in my description of the function?
I kinda liked how impersonal it was.

Reply:
I had to read it three times and now I have a headache.. 😖  😆

Reply:
```suggestion
// Check that requested compute resources are no more than what is listed as available with max_cpus, max_memory and max_time
```